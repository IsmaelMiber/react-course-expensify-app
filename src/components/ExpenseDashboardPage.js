import React from 'react';
import ExpensesList from './ExpensesList';
import ExpensesListFilters from './ExpensesListFilters';

const ExpenseDashboardPage = () => (
    <div>
        This is my ExpenseDashboardPage component
        <ExpensesListFilters />
        <ExpensesList />
    </div>
);

export default ExpenseDashboardPage;