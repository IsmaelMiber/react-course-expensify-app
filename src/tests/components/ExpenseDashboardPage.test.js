import React from 'react';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';
import { shallow } from 'enzyme';

test("Should ExpenseDashboardPage render correctly", () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});