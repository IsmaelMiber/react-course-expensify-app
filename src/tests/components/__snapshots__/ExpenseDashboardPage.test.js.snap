// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ExpenseDashboardPage render correctly 1`] = `
<div>
  This is my ExpenseDashboardPage component
  <Connect(ExpensesListFilters) />
  <Connect(ExpensesList) />
</div>
`;
